jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/util/WebSocketConnectionStatistics.java

```diff
@@ -0,0 +1,40 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.websocket.common.util;
+
+import org.eclipse.jetty.io.Connection;
+import org.eclipse.jetty.io.ConnectionStatistics;
+import org.eclipse.jetty.websocket.common.io.AbstractWebSocketConnection;
+
+public class WebSocketConnectionStatistics extends ConnectionStatistics
```

Comment:
I think this could be generalised as we now probably need the same thing for HTTP stats maybe HTTP2 stats etc.

So perhaps put a protected method into ConnectionStatistics that asks if a connection should be included.... perhaps with the default implementation checking in IncludeExcludeSet of classes?

Reply:
Hmm perhaps keep the base class the same for efficiency, but have an IncludeExclude version derived